keps/sig-storage/20190129-csi-skip-attach.md

```diff
@@ -55,6 +55,8 @@ This KEP contains details that are missing from the design proposal.
 * E2E tests:
     * Drivers that don't support attach don't need the external-attacher and can
       mount volumes successfully
+    * Drivers that don't require attach but has missing CSIDriver objects should successfully
```

Comment:
Also external-attacher is not running right?

Reply:
fixed